CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.5.1] - 2022-07-04
```

Comment:
Removing this library should be considered breaking as it will change the serialization of public objects.
(Keepachangelog requires this to be `## [Unreleased]` until the release is ready, to allow for multiple parallel feature branches.)

Reply:
Done, did that out of habbit